packages/zqlite/src/table-source.ts

```diff
@@ -484,7 +484,7 @@ export class TableSource implements Source {
     if (constraint) {
       for (const [key, value] of Object.entries(constraint)) {
         constraints.push(
-          sql`${sql.ident(key)} IS ${toSQLiteType(
+          sql`${sql.ident(key)} = ${toSQLiteType(
```

Comment:
@tantaman this was changed in https://github.com/rocicorp/mono/commit/c72d614afeb5cd24f18b82e082183752c63caa73

without changing it back this test fails

<img width="719" height="468" alt="image" src="https://github.com/user-attachments/assets/52667397-a522-4e89-b007-34afed73502c" />

https://github.com/rocicorp/mono/blob/d8a6f4d59ef5cfee7f654f041ee181e7cda62c76/packages/zql/src/ivm/source.test.ts#L202